eslint.config.js

```diff
@@ -54,7 +54,10 @@ export default [
 	},
 	{
 		plugins: { deprecation },
-		rules: deprecation.configs.recommended.rules,
+		rules: {
+			...deprecation.configs.recommended.rules,
+			'deprecation/deprecation': 'warn',
```

Comment:
Can we fix the deprecation that caused this? Seems like as good a time as any.

Reply:
Tried, but would require refactoring a bunch of our reply API wrappers, since I wasn't careful about using the `ephemeral` bool that's been deprecated, and I still need to learn how to use the fancy `flags` thing they replaced it with :sweat_smile:

Reply:
Part of the problem is that some of them have overlapping behaviors sometimes, and some of that logic might need to be moved into command functions (e.g. if we want to reply "privately", do we mean ephemeral in the channel or a DM? and if a DM, do we fall back to ephemeral if their DMs aren't enabled? These answers might differ depending on the intent of the command. Right now, some of that is handled with booleans we add to the message option, and those should probs be cleaned up)